modules/services/gpg-agent.nix

```diff
@@ -179,7 +179,7 @@ in {
       };
 
       pinentryFlavor = mkOption {
-        type = types.nullOr (types.enum pkgs.pinentry.flavors);
+        type = types.nullOr (types.enum (pkgs.pinentry.flavors ++ [ "mac" ]));
         example = "gnome3";
         default = "gtk2";
```

Comment:
What do you think about setting the default "flavor" to `mac` when `isDarwin` here? I've tried all the `pkgs.pinentry.flavors` on macOS and the experience is pretty terrible, so `pinentry_mac` seems like a sensible default on macOS IMO.

Reply:
Seems reasonable to me!